drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -261,13 +300,105 @@ static const struct sensor_driver_api lis2dh_driver_api = {
 	.channel_get = lis2dh_channel_get,
 };
 
+int lis2dh_reg_init(const struct device *dev)
```

Comment:
why did you create a separate function for this? (so it can be called from the application for something? please specify specific reason in a comment)

Reply:
I isolated it so it could be re-called after a reset as the reset function only resets the sensors registers and not the device binding etc that is initialized at startup. Do you mean comments in code or in the PR?

Reply:
Comment in code